### DISCUSSION TITLE
老项目该如何接入，文档能够增加相关例子吗？

### DISCUSSION THREAD

**Discussion by knight-peter** [jd-opensource/micro-app#105] [Q&A]
如题，一些jQuery 前端项目该如何接入子应用。

**Comment by bailicangdu** [comment]
你们需要兼容ie吗

**Reply by knight-peter** [reply]
是否兼容ie还没定下来。当然能兼容最好。目前是有好几个老项目是easyui做的jquery项目，想知道如何接入。

**Comment by bailicangdu** [comment]
jQuery可以使用

**Reply by knight-peter** [reply]
能够在micro-app/examples/children/里加个demo吗？或者在文档里写下接入方案。

**Reply by bailicangdu** [reply]
以后会增加，目前的精力在功能实现上

**Reply by perchecc** [reply]
您好，现在有demo了吗？目前看 [micro-app](https://github.com/micro-zoe/micro-app)/[dev](https://github.com/micro-zoe/micro-app/tree/dev/dev)/children/里 还是没有兼容老项目作为基座应用的

**Comment by bailicangdu** [comment]
你们项目是spa还是多页面？

**Reply by knight-peter** [reply]
jquery项目吗？那应该是多页面的。其他的都是spa的。

**Reply by bailicangdu** [reply]
单页面和多页面都可以使用，多页面的方式不同

**Comment by em2046** [comment]
https://gitee.com/liuge1988/kitty-ui
借楼请教一下，目前前端使用这个项目，是多TAB页面的SPA，能够迁移到微前端吗？想迁移后维持之前的用户界面不变，后续能够新增一些vue3的或者react的tab页面。与当前的vue2tab页面共存，SPA的tab页面切换可以使用keep alive保存输入框input中的值。请问微前端能实现吗？

**Reply by bailicangdu** [reply]
可以使用微前端但keep alive不支持
